Dockerfile.operator

```diff
@@ -37,12 +37,17 @@ RUN exportOrFail() { echo $1; if [[ $1 == *= ]]; then echo "Error: empty variabl
               -X ${VERSION_PKG}.autoInstrumentationApacheHttpd=${AUTO_INSTRUMENTATION_APACHE_HTTPD_VERSION} \
               -X ${VERSION_PKG}.autoInstrumentationNginx=${AUTO_INSTRUMENTATION_NGINX_VERSION}"
 
-FROM registry.redhat.io/ubi8/ubi-minimal:latest@sha256:33161cf5ec11ea13bfe60cad64f56a3aa4d893852e8ec44b2fd2a6b40cc38539
+
+FROM registry.redhat.io/ubi8/ubi:latest as ubi-micro-additional-packages
+RUN mkdir -p /mnt/rootfs
+RUN dnf install --installroot /mnt/rootfs --releasever 8 --setopt install_weak_deps=false --nodocs -y coreutils-single glibc-minimal-langpack openssl; dnf clean all
```

Comment:
Also, I'm not sure if `ca-certificates` get automatically pulled in. I'd suggest to double-check and install if needed.